source/cli.ts

```diff
@@ -45,13 +48,14 @@ const cli = meow(`
 		const cwd = cli.input.length > 0 ? cli.input[0] : process.cwd();
 		const typingsFile = cli.flags.typings;
 		const testFiles = cli.flags.files;
+		const {showDiff} = cli.flags;
```

Comment:
If we're updating the CLI, may as well consolidate these variables:

```suggestion
		const {typingsFile, testFiles, showDiff} = cli.flags;
```

Reply:
Could also add a description in the help text:

https://github.com/SamVerschueren/tsd/blob/09c0d17365c7585dd8d77f4c0d454d63d0c58e72/source/cli.ts#L16-L18

```
--show-diff    Show type error diffs         [Default: don't show]
```